CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.4.10] - 2024-12-03
```

Comment:
same numbering as changes in scale up PR

Reply:
True, depending on which on merges first, I'll change the other. Will also result in a merge conflict anyways